biryani/baseconv.py

```diff
@@ -578,6 +578,8 @@ def make_anything_to_float(accept_expression = False):
     (42.0, None)
     >>> make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1) * 42 - 42')
     (42.0, None)
+    >>> make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1)\n* 42 - 42')
```

Comment:
Great PR 🙌 
Sadly the doctstrings tests don't pass, due to the fact that they are strings, and therefore, they see `/n`as a newline in the docstring.
In order for this docstring to pass, you can write : 
`make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1)\\n* 42 - 42')`

Reply:
Thanks @Anna-Livia. Can you explicit the command you run that is failing?

_This repository misses documentation._

Reply:
In a virtual env with after `pip install --editable .[dev]`, `make test` is failing a lot.

Reply:
Merci thanks @Anna-Livia for those details!